src/components/AppProvider/AppProvider.scss

```diff
@@ -17,7 +17,7 @@ body {
 html,
 body,
 button {
-  font-family: font-family();
+  font-family: var(--p-font-family-base);
 }
```

Comment:
@lgriffee we have to be careful here as I a don't think the theme provider will be providing css custom properties at this level. @aaronccasanova and @kyledurand can confirm.

Reply:
This should be fine as the `CustomProperties` component [applies these vars directly to the `:root` pseudo class](https://github.com/Shopify/polaris-react/blob/be9a73d72415aec487a190956e144d4a59f05843/src/components/CustomProperties/styles.ts#L84). The important distinction is that these custom properties are considered static (aka. color-scheme independent). This is a great callout though because if this property was not static, it would need to be a descendent of an element with our  custom `color-scheme` attribute applied (e.g. a child of the `CustomProperties` component)